liquibase-core/src/main/java/liquibase/hub/HubUpdater.java

```diff
@@ -169,7 +167,11 @@ public void postUpdateHub(Operation updateOperation, BufferedLogService bufferLo
 
             //
             // Send the COMPLETE operation event
+            // Capture the current log level to use for filtering
             //
+            java.util.logging.Logger liquibaseLogger = java.util.logging.Logger.getLogger("liquibase");
```

Comment:
The logger people use is pluggable, so not necessarily the java logger like this.

Reply:
I found an example in the code where we were still doing this, I thought it was plausible.